gradoop-flink/src/main/java/org/gradoop/flink/model/api/operators/MultiGraphToGraphOperator.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright © 2014 - 2017 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.model.api.operators;
+
+import org.gradoop.flink.model.api.epgm.LogicalGraph;
+
+/**
+ * Creates a {@link LogicalGraph} based on an at least one input graph.
+ */
+public interface MultiGraphToGraphOperator extends Operator {
```

Comment:
maybe `MultipleGraphToGraphOperator` or `GraphsToGraphOperator` is a better name since `MultiGraph` is a graph that supports parallel edges, i.e. multiple edges between the same two vertices.

Reply:
a very good remark. @p-f please provide this fix :)